src/main/resources/static/user.js

```diff
@@ -56,7 +71,7 @@ function generateSurveyForm(data) {
             <div class="col-sm-8 p-5 rounded">
                 <p><strong>${question.question}</strong></p>
                 <div>
-                    <input type="text" class="form-control" id="id3" placeholder="Enter Number">
+                    <input type="number" min="${question.lowerBound}" max="${question.upperBound}" class="form-control" id="${question.id}" name="${question.id}" placeholder="Enter Number" required>
```

Comment:
this will only be supported if the browser supports html 5

Reply:
all browers support html5 if i am not wrong